src/Core/Components/NavMenu/FluentNavBase.cs

```diff
@@ -98,9 +87,33 @@ public abstract class FluentNavBase : FluentComponentBase
     [Parameter]
     public bool ForceLoad { get; set; }
 
+    /// <summary>
+    /// Gets or sets the id of the custom toggle element
+    /// Defaults to navmenu-toggle
+    /// </summary>
+    [Parameter]
+    public string CustomToggleId { get; set; } = "navmenu-toggle";
```

Comment:
1. How to be sure that multiple Navmenu will have a unique `CustomToggleId`?
1. Why not to use the `Id` + `toggle` string to have a unique ID?

Reply:
The 'happy path' is that there will be 1 `FluentNavMenu` in a site. In that case you don't need to set the id and the defaults will 'just work'.
In case someone wants to use multiple nav menu's in one site **and** also wants to make them all toggle-able, they will need to supply these id's and make sure they are unique,